broker-cli/commands/wallet.js

```diff
@@ -241,6 +241,38 @@ const NETWORK_STATUSES = Object.freeze({
   INACTIVE: 'INACTIVE'
 })
 
+/**
+ * Formats the capacities to fixed numbers and colors them if they are greater
+ * than 0. Will return Not Available if balance is empty
+ *
+ * @function
+ * @param {String} balance
+ * @param {String} status
+ * @return {String} balance - formatted with size and color
+ */
+function formatBalance (balance, status) {
+  if (!balance) {
```

Comment:
I'm not sure we want to use the balance not existing as evidence that the engine is unavailable - any way to make this more explicit.

If we are to keep it this way, I'd prefer to do something like `balance === undefined`

Reply:
@treygriffith it will come across as an empty string through gRPC. Unfortunately there is not a pretty way to do this. The code that uses `formatBalance` is massive.